src/Components/Authentication/FormSwitcher.tsx

```diff
@@ -18,6 +18,7 @@ import {
 } from "./Types"
 
 export interface FormSwitcherProps {
+  contextModule?: string
```

Comment:
Shouldn't `contextModule` come from `ModalOptions`?

Reply:
Yeah this isn't actually needed. I'm cleaning this up a bit.